setup.py

```diff
@@ -1,7 +1,7 @@
 from setuptools import setup, find_packages
 
 __package_name__ = "validations_engine"
-__version__ = "1.0.0"
```

Comment:
I suggest you open separate PRs: one (or more with the changes) and one only with the release.
This keeps things more organized.
I.g.: https://github.com/quintoandar/validations-engine/pull/9
Also remember to update the changelog.